jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildDockerMojo.java

```diff
@@ -44,7 +44,6 @@
     name = BuildDockerMojo.GOAL_NAME,
     requiresDependencyResolution = ResolutionScope.RUNTIME_PLUS_SYSTEM)
 public class BuildDockerMojo extends JibPluginConfiguration {
-
```

Comment:
nit: a line removed (we're following a convention of empty line after class definition for this codebase)